api/controllers/v0.3/IlsClient.js

```diff
@@ -317,6 +401,9 @@ IlsClient.ADULT_METRO_PTYPE = 10;
 IlsClient.ADULT_NYS_PTYPE = 11;
 IlsClient.SENIOR_METRO_PTYPE = 20;
 IlsClient.SENIOR_NYS_PTYPE = 21;
+IlsClient.SIMPLYE_JUVENILE = 4;
```

Comment:
What's the difference between "juvenile" and "juvenile only"?

Reply:
I'm not sure. @killerpuppytails ?

Reply:
Juvenile means all access - it doesn't have any restrictions for checkout. The reason we have it is to allow appropriate database service access. Juvenile only does have restrictions for checkout.